### PR TITLE
also log screenshot and full HTML for failed e2e

### PR DESCRIPTION
**Pull request by superyyrrzz**

### CODE REVIEW

File: test/docfx.E2E.Tests/DocfxSeedSiteTest.cs [hunk 1]
```diff
@@ -114,8 +115,9 @@ public void TestReferencePage()
 
             // check overwrite
             var conceptual = _driver.FindElement(By.ClassName("conceptual"));
-            // Add this line to help find out what sometime breaks e2e
-            Assert.True(conceptual.Text.Contains("This is a class talking about CAT."), $"Actual HTML: {conceptual.GetAttribute("outerHTML")}");
+            // Add these lines to help find out what sometime breaks e2e
```
Comment (hellosnow) [inline test/docfx.E2E.Tests/DocfxSeedSiteTest.cs]: >help find [](start = 34, length = 9)

minor: help finding out?

**Comment by hellosnow** [comment]
:shipit:

File: test/docfx.E2E.Tests/DocfxSeedSiteTest.cs (hunk 1 above)
Comment (superyyrrzz) [inline test/docfx.E2E.Tests/DocfxSeedSiteTest.cs]: "help (to) do" is also right

Comment (hellosnow) [inline test/docfx.E2E.Tests/DocfxSeedSiteTest.cs]: yes you're right, to could be missed

---
In reply to: [92092186](https://github.com/dotnet/docfx/pull/997#discussion_r92092186) [](ancestors = 92092186)

**Pull request merged**